examples/05_stable_diffusion/src/modeling/unet_2d_condition.py

```diff
@@ -81,6 +82,7 @@ def __init__(
         norm_eps: float = 1e-5,
         cross_attention_dim: int = 1280,
         attention_head_dim: Union[int, Tuple[int]] = 8,
+        use_linear_projection = False,
```

Comment:
Please remove the spaces around `=`.

Reply:
changed to `use_linear_projection: bool = False,` to make it similar to other bool params in this `__init__` function.